ecosystem/gstreamer_plugin/gst_mtl_st40_rx.c

```diff
@@ -361,7 +361,9 @@ static GstFlowReturn gst_mtl_st40_rx_fill_buffer(Gst_Mtl_St40_Rx* src, GstBuffer
 
   hdr = (struct st40_rfc8331_rtp_hdr*)usrptr;
   payload_hdr = (struct st40_rfc8331_payload_hdr*)(&hdr[1]);
+  payload_hdr->swaped_second_hdr_chunk = ntohl(payload_hdr->swaped_second_hdr_chunk);
```

Comment:
Good catch but with this the size of udw changes all the time please change the warning at line 375 into INFO or DEBUG 

I personally think INFO would be best 

![image](https://github.com/user-attachments/assets/1bc6b3a7-0344-4ff3-ac86-3bd229550c8c)